### DISCUSSION TITLE
Problems with the Skia.GTK runtime

### DISCUSSION THREAD

**Discussion by hui3215** [unoplatform/uno#9408] [Q&A]
When running with Skia.GTK, my program had a problem, but the program still worked, but the animation effects were gone, and everything became very blunt
![image](https://user-images.githubusercontent.com/53279456/182277688-400e5c48-46c5-4b47-b24d-ff81a4ef6453.png)

**Comment by jeromelaban** [comment]
Thanks for the report. What do you mean by "became blunt" ? Would you be able to create a video of the issue?

**Reply by hui3215** [reply]
That is, some have no animation effect, and some animation effects are very stuttering
